lib/jsdom/living/helpers/focusing.js

```diff
@@ -60,6 +61,12 @@ exports.isFocusableAreaElement = elImpl => {
     // attribute is not implemented.
   }
 
+  if (elImpl._namespaceURI === SVG_NS) {
```

Comment:
This should probably be an `else if`, since the namespace URI can’t be both the `HTML_NS` and the `SVG_NS`.

```suggestion
  } else if (elImpl._namespaceURI === SVG_NS) {
```

Reply:
Thanks for reminding.
I return the `false` in advance so that others don’t have to read the code behind to know the return value when `elImpl._namespaceURI` is `HTML_NS`.